eclipse-scout-core/src/table/TableHeader.ts

```diff
@@ -456,7 +456,7 @@ export class TableHeader extends Widget implements TableHeaderModel {
     let accessibleStateText = '';
     if (column.sortActive) {
       let sortDirection = column.sortAscending ? 'asc' : 'desc';
-      accessibleStateText += this.session.text('ui.Sorting') + ' ' + (column.sortAscending ? this.session.text('ui.ascending') : this.session.text('ui.descending'));
+      accessibleStateText += ' ' + this.session.text('ui.Sorting') + ' ' + (column.sortAscending ? this.session.text('ui.ascending') : this.session.text('ui.descending'));
```

Comment:
removed, spec changed to test `${column1.text}sorting ascending`.

Screen reader does not need it, it will treat each DOM node as its own navigatable text. This may or may not change if the table header ever becomes tabbable.